### PR TITLE
[13.0] Use stock_helper to replace duplicated implementation

### PR DESCRIPTION
**Pull request by guewen**
See https://github.com/OCA/stock-logistics-warehouse/pull/1073

This is for:

- wms/stock_dynamic_routing
- wms/shopfloor
- wms/stock_move_source_relocate

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 freezegun
+odoo13-addon-stock_helper @ git+https://github.com/OCA/stock-logistics-warehouse@refs/pull/1073/head#subdirectory=setup/stock_helper
```
Comment (sebalix) [inline test-requirements.txt]: To remove now it has been merged

**Review by sebalix — COMMENTED** [review]
`stock_move_source_relocate` module should depend on `stock_helper` too

File: test-requirements.txt (hunk 1 above)
Comment (sebalix) [reply test-requirements.txt]: Fixed

**Comment by sebalix** [comment]
I pushed again to remove the test dependency commit now that OCA/stock-logistics-warehouse#1073 is merged + fixed the missing dependency for `stock_move_source_relocate`.

**Review by sebalix — APPROVED** [review]

**Comment by simahawk** [comment]
Please wait to merge this. I'll rebase it on top of #169 once is merged.

**Comment by simahawk** [comment]
rebased

**Pull request merged**